### ISSUE TITLE
Reconsider git ls-files | grep "\.v$" >> _CoqProject

### ISSUE THREAD

**Issue by satnam6502** [project-oak/silveroak#472]
I am not sure if we want to have the Coq source files discovered in this way:
```
git ls-files | grep "\.v$" >> _CoqProject
```
I often trip up when added a new file which has not had `git add` and commit applied to it.
When we have some stability perhaps it is not too bad just to explicitly list the source files in `_CoqProject`.

I also deleted a file `Types.v` but this still gets filtered to `_CoqProject`.
```
$ git status
On branch notypes
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   Cava/Cava.v
	modified:   Cava/Extraction.v
	deleted:    Cava/Types.v
	modified:   Cava2HDL.cabal
$ git ls-files | grep "\.v$"
...
Cava/Types.v
...
$ make
git ls-files | grep "\.v$" >> _CoqProject
coq_makefile -f _CoqProject -o Makefile.coq
/Library/Developer/CommandLineTools/usr/bin/make -f Makefile.coq
COQC Cava/Acorn/Acorn.v
File "./Cava/Acorn/Acorn.v", line 17, characters 15-35:
Error: Unable to locate library Cava.Acorn.CavaClass.

make[2]: *** [Cava/Acorn/Acorn.vo] Error 1
make[1]: *** [all] Error 2
make: *** [coq] Error 2
```

If I add the changes then things work fine.

CC: @jadephilipoom

**Comment by jadephilipoom** [comment]
I'm fine with making this change; `git ls-files` is just a default I carried with me from a previous project. I'd prefer not to list everything manually in `_CoqProject`, though, because the reason I made the change is that we had multiple instances of a file accidentally not getting added and therefore not being checked by CI. However, listing all `.v` files whether or not they're known to git seems like a reasonable solution.
